terraform/modules/gost_website/tpl/deploy-config.js

```diff
@@ -1,2 +1,3 @@
 window.APP_CONFIG = window.APP_CONFIG || {};
 window.apiURLForGOST = 'https://${gost_api_domain}/';
+window.UseNewTable = ${use_new_table};
```

Comment:
This needs to be added to `window.APP_CONFIG` in order to be consistent with the way it's being referenced in the `.vue` files. Note that you _could_ keep this line unmodified and update those references to `window.UseNewTable`, but my personal preference would be to avoid cluttering the top-level `window` namespace and use `APP_CONFIG` for these types of things.

```suggestion
window.APP_CONFIG.UseNewTable = ${use_new_table};
```